src/Orchestration/Adapter/DockerAPI.php

```diff
@@ -275,29 +279,64 @@ public function getStats(string $container = null, array $filters = []): array
         foreach ($containerIds as $containerId) {
             $result = $this->call('http://localhost/containers/'.$containerId.'/stats?stream=false', 'GET');
 
-            if ($result['code'] !== 200) {
+            if ($result['code'] !== 200 || empty($result['response'])) {
                 throw new Orchestration($result['response']);
```

Comment:
In case of when `empty($result['response']) === true`, then nothing will be thrown?

Reply:
Simplified: `if (empty($result['response'])) { throw;`

If response is empty, error is thrown

Reply:
I can't see any problem either, this was added as sometimes the docker API had this bug and sent an empty body, and we can't proceed.